### ISSUE TITLE
Problem with logging

### ISSUE THREAD

**Issue by tvillingett** [naofireblade/homebridge-weather-station-extended#22]
There is some problem with the logging, almost nothing gets logged

Here is the config:
```
{
	"platform": "WeatherStation",
	"name": "Weather Station",
	"key": "358b48f5b44b2c8c",
	"location": "OSD"
}
```

And below is the logs.
```
20:47:59 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
20:47:59 - [Weather Station] Exception while parsing current observations: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: cd65a9ab-85ad-494a-b2bd-2f380084134d
20:47:59 - [Weather Station] Exception while parsing forecast for day #0: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:47:59 - [Weather Station] Exception while parsing forecast for day #1: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:47:59 - [Weather Station] Exception while parsing forecast for day #2: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:47:59 - [Weather Station] Exception while parsing forecast for day #3: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:51:58 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
20:51:58 - [Weather Station] Exception while parsing current observations: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: cd65a9ab-85ad-494a-b2bd-2f380084134d
20:51:58 - [Weather Station] Exception while parsing forecast for day #0: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:51:58 - [Weather Station] Exception while parsing forecast for day #1: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:51:58 - [Weather Station] Exception while parsing forecast for day #2: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:51:58 - [Weather Station] Exception while parsing forecast for day #3: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:55:59 - [Weather Station] Exception while parsing current observations: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: cd65a9ab-85ad-494a-b2bd-2f380084134d
20:55:59 - [Weather Station] Exception while parsing forecast for day #2: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:55:59 - [Weather Station] Exception while parsing forecast for day #1: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:55:59 - [Weather Station] Exception while parsing forecast for day #0: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
20:55:59 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
```

**Comment by naofireblade** [comment]
Hi, that looks strange. Did you changed anything in the code? The output of your log (adding the same characteristic twice usally cannot happen).

**Comment by tvillingett** [comment]
No, I haven't made any changes, it is there original code. Is there any debug info needed to know more?

**Comment by tvillingett** [comment]
I just upgraded to ver 1.2.3, but no difference:
`10:37:26 - Registering platform 'homebridge-alarmdotcom.Alarmdotcom'
10:37:26 - Registering accessory 'homebridge-cmd.CMD'
10:37:26 - Registering platform 'homebridge-config-ui-x.config'
10:37:26 - Registering platform 'homebridge-controllerlink.HomeBridgeControllerLink'
10:37:26 - Registering accessory 'homebridge-daylight.Daylight'
10:37:26 - Registering accessory 'homebridge-temperature-ds18b20.DS18B20'
10:37:26 - Registering accessory 'homebridge-efergy.Efergy'
10:37:26 - homebridge API version: 2.2
10:37:26 - Registering platform 'homebridge-mqtt.mqtt'
10:37:26 - Registering accessory 'homebridge-mqtt-power-consumption.mqtt-power-consumption'
10:37:26 - Registering accessory 'homebridge-mqtt-temperature.mqtt-temperature'
10:37:26 - Registering accessory 'homebridge-mqttgarage.Mqttgarage'
10:37:26 - Registering accessory 'homebridge-mqttswitch.mqttswitch'
10:37:26 - Registering accessory 'homebridge-ninjablock-alarmstatedevice.NinjaBlock-AlarmStateDevice'
10:37:26 - Registering accessory 'homebridge-people.people'
10:37:26 - Registering platform 'homebridge-telldus.Telldus'
10:37:26 - Registering platform 'homebridge-wunderground-extended.WeatherStation'
10:37:26 - Registering accessory 'homebridge-wunderground.WUNDERGROUND'
10:40:05 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
10:40:05 - [Weather Station] Exception while parsing current observations: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: cd65a9ab-85ad-494a-b2bd-2f380084134d
10:40:05 - [Weather Station] Exception while parsing forecast for day #0: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:40:05 - [Weather Station] Exception while parsing forecast for day #1: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:40:05 - [Weather Station] Exception while parsing forecast for day #2: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:40:05 - [Weather Station] Exception while parsing forecast for day #3: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:44:03 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
10:44:03 - [Weather Station] Exception while parsing current observations: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: cd65a9ab-85ad-494a-b2bd-2f380084134d
10:44:03 - [Weather Station] Exception while parsing forecast for day #0: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:44:03 - [Weather Station] Exception while parsing forecast for day #1: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:44:04 - [Weather Station] Exception while parsing forecast for day #2: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
10:44:04 - HAP Warning: Characteristic undefined not in required or optional characteristics for service 0000008A-0000-1000-8000-0026BB765291. Adding anyway.
10:44:04 - [Weather Station] Exception while parsing forecast for day #3: Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: 57f1d4b2-0e7e-4307-95b5-808750e2c1c7
`

**Comment by naofireblade** [comment]
I see that you are also using homebridge-wunderground. Maybe there is an incompatiblity. Can you disable that plugin for testing purposes? Also do you use more than one homebridge instance? Do you use the default config for homebridge-weather-station-extended or did you set the forecast parameter?

**Comment by tvillingett** [comment]
I looked again now in the app, and it has diagram working from 11.30 today (I didn't look until after i installed 11.2.6 so it might also helped?)

**Comment by naofireblade** [comment]
The iOS update cannot be the solution because the erros you posted occured in homebridge and not in iOS. I still guess there is an unusual setup on your side like more than one instance of homebridge.
